configs/package.json

```
@@ -0,0 +1,33 @@
+{
+  "name": "@aragon/osx-commons-configs",
+  "author": "Aragon Association",
+  "version": "1.0.0",
+  "license": "MIT",
```

```suggestion
  "license": "AGPL-3.0-or-later",
```
We use AGPL-3.0 or later everywhere

---

done